guides/plugins/plugins/bundle.md

```diff
@@ -0,0 +1,109 @@
+---
+nav:
+  title: Bundle
+  position: 20
+
+---
+
+# Bundle
+
+Plugins are based on the Symfony bundle concept, but offer additional features like lifecycle events and the ability to be managed in the Shopware administration. This is maybe unwanted in some cases, like project critical customizations which should not be managed via the Shopware administration. In this case, you can use a Symfony bundle instead of a plugin.
+
+The Bundle can be put into the `src/` folder of your project and needs to be registered in the `config/bundles.php` file of your project.
+
+## Choosing the right Bundle class
+
+There are two Bundle classes you can choose from:
+
+- `Shopware\Core\Framework\Bundle`
+- `Symfony\Component\HttpKernel\Bundle\Bundle`
+
+The first one is the Shopware bundle class and the second one is the Symfony bundle class. The Shopware bundle class extends the Symfony bundle class, but offers additional features like to be an Theme, bring JavaScript/CSS files, Migrations, etc. If you don't need these features, you can use the Symfony bundle class instead.
+
+## Creating a Bundle
+
+In the Shopware project template it is intended to put the bundle into the `src/` folder of your project and use the namespace `App\`.
```

Comment:
was already mentioned above with the src directory. 

and does it really always need to be `App` ? :thinking: